show/main.py

```diff
@@ -520,6 +741,52 @@ def uptime():
     """Show system uptime"""
     run_command('uptime -p')
 
+@cli.command()
+def clock():
+    """Show date and time"""
+    run_command('date')
+
+@cli.command('system-memory')
+def system_memory():
+    """Show memory information"""
+    comm="free -m"
+    run_command(comm)
+
+@click.group(cls=AliasedGroup, default_if_no_args=False)
+def vlan():
+    """Show VLAN information"""
+    pass
+
+cli.add_command(vlan)
+
+@vlan.command()
+def brief():
+    """Show all bridge information"""
+    comm="sudo brctl show"
```

Comment:
Change to `command = "..."` for consistency ("comm" -> "command" and add spaces around equals sign.